common/remote/sockfile.cpp

```diff
@@ -3966,8 +4003,7 @@ class CRemoteDiskBaseActivity : public CSimpleInterfaceOf<IRemoteActivity>, impl
 //interface IVirtualFieldCallback
     virtual const char * queryLogicalFilename(const void * row) override
     {
-        const char * filename = nullptr; // MORE: This needs to be passed from the client to dafilesrv
-        return filename ? filename : "";
+        return logicalFilename.length() ? logicalFilename.get() : "";
```

Comment:
picky: logicalFilename.get() ? ... is better.  length will perform a strlen() each call.

Reply:
actually, 
    return logicalFilename.str();
is even simpler.

Reply:
agreed, will change.